be1-go/cli/witness/witness.go

```diff
@@ -39,19 +42,32 @@ func WitnessServe(context *cli.Context) error {
 
 	h := hub.NewWitnessHub(point)
 
-	var addr = flag.String("addr", "localhost:8080", "http service address")
-	u := url.URL{Scheme: "ws", Host: *addr, Path: "/echo"}
-	var ws *websocket.Conn
-	ws, _, err = websocket.DefaultDialer.Dial(u.String(), nil)
-
+	ws, err := witConnectToOrganizer(orgIpAddr, port)
 	if err != nil {
 		return xerrors.Errorf("failed to connect to organizer: %v", err)
 	}
 
 	organizerSocket := hub.NewOrganizerSocket(h, ws)
 
-	organizerSocket.WritePump()
-	organizerSocket.ReadPump()
+	go organizerSocket.WritePump()
+	go organizerSocket.ReadPump()
+
+	done := make(chan struct{})
+	h.Start(done)
+
+	//TODO: connect to clients and other witnesses
+
+	done <- struct{}{}
```

Comment:
Same here, there must a good reason I just don't see it.

Reply:
Same as above for witness.